DuckDuckGo/MainViewController.swift

```diff
@@ -64,6 +64,25 @@ class MainViewController: UIViewController {
         }
     }
     
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        showiOS11Popup()
```

Comment:
No need to show this to iOS 10< users, could wrap it in 

`if #available(iOS 11.0, *) {`

If so, you probably want to update the copy of your alert message as well.

Reply:
Done! Good idea.